test/system/experimental.test.ts

```diff
@@ -82,3 +82,54 @@ test("docker-archive image type throws on bad files", async (t) => {
     "throws when the provided path is a directory",
   );
 });
+
+test("static scan for Identifier type image (nginx:1.19.0)", async (t) => {
```

Comment:
why is this test part of this PR?

Reply:
There is no test for the flow without `docker-archive` or `oci-archive`. I created one to help me to follow the flow and include the option. It's giving me confidence that I'm not broken nothing.
Maybe there is this test in another place and I missed it. If it already exists I can delete this test.